tournesolbot/__main__.py

```diff
@@ -0,0 +1,281 @@
+import sys
+import requests
+import pandas as pd
+import getopt
+
+from tournesolbot.tournesol_api_functions import get_good_video, get_video_info, get_missing_channel_list
+from tournesolbot.twitter_api_functions import twitter_authentication, write_tweet, write_response_tweet
+from tournesolbot.data.utils_dict import ACCEPTED_LANGUAGE,YT_2_TWITTER, CRITERIA_DICT
+from tournesolbot.data.utils_dict import already_answered_filepath, daily_tweet_text, video_details_tweet_text, not_found_video_tweet_text
+
+
+# Parameters
+FROM_TOP = 70
+LAST_N_DAYS = 120
+
+def get_top_percentage(criteria_row):
```

Comment:
I completely refactored the function below to use math instead a result of matrix. 

Please have a look to https://github.com/tournesol-app/tournesol-twitter-bot/pull/11/commits/3823c5d44538a116ad1c0c3635552e6cee07a66e

This is why I wrote a sentence about gerrit vs github pull request.